### PR TITLE
send email notifications

### PR DESCRIPTION
**Pull request by simonarcher99**
some code that is

### CODE REVIEW

File: src/blueprints/notifications.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+Good luck!
+"""
+
+@notifications.route('/check_for_predictions', methods=['POST'])
+def check_upcoming_games()
+
+    timezone = pytz.timezone('Europe/London')
+    today = datetime.today()
+    today = datetime(today.year, today.month, today.day)
+    matches = session.query(Match).filter(Match.match_date = today).all()
```
Comment (luke142367) [inline src/blueprints/notifications.py:27]: ==?

**Review by lukeely — APPROVED** [review]

**Comment by lukeely** [comment]
Looks sexy

File: src/blueprints/notifications.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        
+        current_time = datetime.now(timezone)
+
+        if combined < (current_time + datetime.timedelta(hours = 1)) and not check_kicked_off(match.matchid):
+            check_user_prediction(match.matchid)
+
+def check_user_prediction(match):
+    users = session.query(User).filter(User.hidden == False).all()
+    for user in users:
+        if session.query(exists().where(Prediction.userid == userid).where(Prediction.matchid == match.matchid)).scalar():
```
Comment (alicolver) [inline src/blueprints/notifications.py]: ```if not session.query(exists().where(Prediction.userid == userid).where(Prediction.matchid == match.matchid)).scalar():                send_email_reminder(user, match)```

**Comment by lukeely** [comment]
I will try to test this locally and then I will merge it

File: src/blueprints/notifications.py (hunk 2 above)
Comment (lukeely) [reply src/blueprints/notifications.py]: +1

Comment (simonarcher99) [reply src/blueprints/notifications.py]: +2

**Comment by simonarcher99** [comment]
> I will try to test this locally and then I will merge it

cheers let me know if the query doesn't work

**Pull request closed without merging**